### ISSUE TITLE
Port MinMon to wcoss2

### ISSUE THREAD

**Issue by EdwardSafford-NOAA** [NOAA-EMC/GSI#225]
Port MinMon DA package to wcoss2.  

Initially I'd planned to remove the version numbers from the MinMon package (as I've done in the other *Mon packages).  But after re-reading the directions for porting to wcoss2, it's clear that that change does not fall under the directions to make changes only as required for porting to the platform.  So I'll wait on restructuring the package.

**Comment by EdwardSafford-NOAA** [comment]
Another issue -- wcoss2 does not seem to have the perl module List/MoreUtils.pm installed.  I opened ticket with the help desk.

**Comment by EdwardSafford-NOAA** [comment]
No word yet from the wcoss2 help desk.  The missing perl lib is List:MoreUtils which is used by the minmon_xtrct_gnorms.pl script.  I could re-write the script to avoid use the 3 elements from List:MoreUtils, but I won't do that unless that perl lib can't/won't be installed on wcoss2.  It's on wcoss_c, wcoss_d, and hera systems, so it should be part of wcoss2.

The Korn shell issue has been resolved.

@MichaelLueken-NOAA

**Comment by EdwardSafford-NOAA** [comment]
Chris at the helpdesk reported late yesterday that  the missing perl module, List::MoreUtils, has been installed on wcoss2, and directed me to include this in my calling script:

`module load intel perl`

I did so.  Now the job encounters this problem:

`XS.c: loadable library and perl binaries are mismatched (got handshake key 0xed00080, needed 0xde00080)`

Initial investigation suggests that this is caused by this may happen when perl or perl modules are installed separately along side the official packages. And that an environment variable can be used to switch to a different perl installation.  I'm looking into that now.  I also reported this to Chris to see if the help desk has a known solution.

**Comment by EdwardSafford-NOAA** [comment]
Per Chris:

`If you're like me (who uses the system-installed perl exclusively), your 'shebang' will say #!/usr/bin/perl.  The problem will be that the perl version you are calling will be the system perl (version 5.26 built with gcc).  This causes a mismatch, the perl I build is at version 5.32 with various compilers.   You'll need to change your shebang to permit the perl found in the perl module you have loaded.  Please do NOT hard-code the path to the version of perl you find from the module.  Doing so, prevents you (and us) from freely updating the version of perl the module loads as updates occur.  Additionally, our package build tool, spack, embeds a hash value tied to the installation that will change as we perform updates/changes (this is the primary reason not to hard code the path).  Change your 'shebang' to the following (it will take the first version of perl it finds in your path.)

#!/usr/bin/env perl`

This works on wcoss2.  I ran a test on wcoss_d and it works there as well.  I'll clean things up and issue a PR for this later today.

**Comment by EdwardSafford-NOAA** [comment]
PR #264 was merged to branch feature/gfsda.v16.1.5_wcoss2_port.  That resolves this issue.
